rails-time-track/config/routes.rb

```diff
@@ -1,3 +1,5 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  namespace :api do
+    resources :projects
```

Comment:
In this lines we are creating all the CRUD routes (index, show, edit, update, delete), aren't we? Maybe it could be better to specify which routes we are using.
For example: `resources :projects, :only => [:index]`

Reply:
Yes, you right, now I change it :sweat_smile: